5-test/job-yaml.template

```diff
@@ -18,6 +18,8 @@ spec:
       restartPolicy: Never
       containers:
       - name: main
+        # if using pre-built images in public repository, use the following image for x86 based nodes:
```

Comment:
please remove from here, this comment and example values belong in config.properties
registry=public.ecr.aws/a2u7h5w3
test_image_name=test-bert-base-workshop .... please include test in the image name, not in the tag
test_imaage_tag=v10-cpu